src/niveristand/__init__.py

```diff
@@ -4,10 +4,14 @@
 from __future__ import unicode_literals
 from niveristand._decorators import nivs_rt_sequence, NivsParam
 from niveristand.realtimesequencetools import run_py_as_rtseq, save_py_as_rtseq
+from ._auto_generated_classes import ErrorCode, VeriStandSdfError, XMLVersionInfo
```

Comment:
I'd like to understand whether we should be using `niveristand.foo` or `.foo` when importing. At a minimum it seems like we should be consistent?

Reply:
For consistency at the top level here I've changed it to be `niveristand._auto_generated_classes`. For the submodules and their auto-generated headers, I think it's fine to leave it as relative imports unless you want me to tweak the code generation itself.